code/model/modelread.cpp

```diff
@@ -1240,6 +1233,14 @@ int read_model_file(polymodel * pm, char *filename, int n_subsystems, model_subs
 					pm->submodel[n].dumb_turn_rate = 0.0f;
 				}
 
+				// Sets can_move on submodels which are of a rotating type or which have such a parent somewhere down the hierarchy
+				if ( (pm->submodel[n].movement_type != MOVEMENT_TYPE_NONE)
+					|| strstr(props, "$triggered:") || strstr(props, "$rotate") || strstr(props, "$gun_rotation:") || strstr(props, "$gun_rotation") ) {
```

Comment:
I suppose the `$gun_rotation:` check with the is redundant with the `$gun_rotation` check, but I intentionally removed the `$dumb_rotate` check because it's made irrelevant by moving this check to below where `$dumb_rotate` is handled.